src/components/Sidebar.tsx

```
@@ -1,4 +1,14 @@
-import { Map, FileText, Plane, LogIn, Heart, MessageSquare, Sparkles, ChevronLeft, ChevronRight } from 'lucide-react';
+import {
+  Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)